### PR TITLE
define ls8 pixel count product definition - draft

### PR DESCRIPTION
**Pull request by NikitaGandhi**
**draft:** define ls8 pixel count product definition

### CODE REVIEW

**Review by fangfy — APPROVED** [review]

File: products/pc_ls8_annual.odc-product.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+metadata:
+  product:
+    name: pc_ls8_annual
+
+storage:
+  crs: EPSG:6933
+  resolution:
+    x: 20
+    y: -20
```
Comment (alexgleith) [inline products/pc_ls8_annual.odc-product.yaml:17]: I think it's 30 m.

**Review by alexgleith — CHANGES_REQUESTED** [review]
Landsat's PIXEL_QA band is 30 m

File: products/pc_ls8_annual.odc-product.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    name: pc_ls8_annual
+
+storage:
+  crs: EPSG:6933
+  resolution:
+    x: 30
+    y: -30
+  tile_size:
+    x: 96000
+    y: 96000
```
Comment (alexgleith) [inline products/pc_ls8_annual.odc-product.yaml:20]: Is this accurate actually?

You can run `rio info <file>` on one of the output files...

Comment (NikitaGandhi) [reply products/pc_ls8_annual.odc-product.yaml]: thanks Alex, based on your input I changed it's value to x: 3200, y:3200. hope this is write.

```
rio info x180/y084/2018--P1Y/pc_ls8_annual_x180y084_2018--P1Y_clear.tif
{"blockxsize": 800, "blockysize": 800, "bounds": [-96000.0, 672000.0, 0.0, 768000.0], "colorinterp": ["gray"], "compress": "deflate", "count": 1, "crs": "EPSG:6933", "descriptions": [null], "driver": "GTiff", "dtype": "uint16", "height": 3200, "indexes": [1], "interleave": "band", "lnglat": [-0.4974800549382149, 5.652736058193159], "mask_flags": [["all_valid"]], "nodata": null, "res": [30.0, 30.0], "shape": [3200, 3200], "tiled": true, "transform": [30.0, 0.0, -96000.0, 0.0, -30.0, 768000.0, 0.0, 0.0, 1.0], "units": [null], "width": 3200}
```

Comment (alexgleith) [reply products/pc_ls8_annual.odc-product.yaml]: Looks correct!

**Pull request merged**